[2022-12-07T07:17:16.682+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag.pull_data manual__2022-12-07T07:17:09.003141+00:00 [queued]>
[2022-12-07T07:17:16.697+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag.pull_data manual__2022-12-07T07:17:09.003141+00:00 [queued]>
[2022-12-07T07:17:16.698+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-12-07T07:17:16.699+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-12-07T07:17:16.700+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-12-07T07:17:16.720+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): pull_data> on 2022-12-07 07:17:09.003141+00:00
[2022-12-07T07:17:16.725+0000] {standard_task_runner.py:55} INFO - Started process 27925 to run task
[2022-12-07T07:17:16.729+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag', 'pull_data', 'manual__2022-12-07T07:17:09.003141+00:00', '--job-id', '1446', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp49h3zzeg']
[2022-12-07T07:17:16.730+0000] {standard_task_runner.py:83} INFO - Job 1446: Subtask pull_data
[2022-12-07T07:17:16.913+0000] {task_command.py:376} INFO - Running <TaskInstance: dag.pull_data manual__2022-12-07T07:17:09.003141+00:00 [running]> on host 505aa50095aa
[2022-12-07T07:17:17.045+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=lofitravel
AIRFLOW_CTX_DAG_ID=dag
AIRFLOW_CTX_TASK_ID=pull_data
AIRFLOW_CTX_EXECUTION_DATE=2022-12-07T07:17:09.003141+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-07T07:17:09.003141+00:00
[2022-12-07T07:17:17.068+0000] {logging_mixin.py:137} INFO - [['2022-12-07', 'USD', 'EUR', 0.95701101], ['2022-12-07', 'USD', 'RUB', 63.0055863], ['2022-12-07', 'EUR', 'RUB', 65.83580088]]
[2022-12-07T07:17:17.079+0000] {logging_mixin.py:137} INFO - Postgresql connection created!
[2022-12-07T07:17:17.080+0000] {logging_mixin.py:137} INFO - Failed to insert into PostgreSQL
[2022-12-07T07:17:17.081+0000] {logging_mixin.py:137} INFO - unsupported format character 'x' (0x78) at index 35
[2022-12-07T07:17:17.082+0000] {logging_mixin.py:137} INFO - CSSS
[2022-12-07T07:17:17.082+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-12-07T07:17:17.101+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=dag, task_id=pull_data, execution_date=20221207T071709, start_date=20221207T071716, end_date=20221207T071717
[2022-12-07T07:17:17.142+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-07T07:17:17.166+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
